src/test/resources/checkstyle.xml

```diff
@@ -235,10 +235,6 @@ Change log:
         </module>
         <module name="NonEmptyAtclauseDescription"/>
         <module name="JavadocTagContinuationIndentation"/>
-        <module name="SummaryJavadoc">
```

Comment:
why remove this?

Reply:
I find for many methods, this leads to redundant code. For basic functions, a javadoc 
```
@returns an object transformed from input with blah blah behavior
```

seems totally sufficient. By requiring both a description and a returns flag, there is often comment duplication. For functions doing interesting things, sure add both a description, and param/return snippets, but don't enforce it an all methods is my opinion.